[Component Governance] Explicit dependency Portable.BouncyCastle added.

dannyjdev opened the pull request:
Build verified that this version removes the alert.

agr commented on tests/Validation.PackageSigning.Core.Tests/Validation.PackageSigning.Core.Tests.csproj:
```diff
@@ -71,6 +71,10 @@
     <PackageReference Include="Newtonsoft.Json">
       <Version>12.0.2</Version>
     </PackageReference>
+    <!-- Explicit dependency added for component governance; Root dependency is Test.Utility -->
```
What is `Test.Utility`? Can we update the reference there instead?

agr commented on tests/Validation.PackageSigning.Core.Tests/Validation.PackageSigning.Core.Tests.csproj:
```diff
@@ -71,6 +71,10 @@
     <PackageReference Include="Newtonsoft.Json">
       <Version>12.0.2</Version>
     </PackageReference>
+    <!-- Explicit dependency added for component governance; Root dependency is Test.Utility -->
```
Is it client's [Test.Utility](https://github.com/NuGet/NuGet.Client/tree/dev/test/TestUtilities/Test.Utility)? Shouldn't it be fixed there then?

agr replied:
It seems they are already on 1.8.10 there. Should we just update our version?

agr reviewed (approved):
Please see if we can simply reference newer `Test.Utility` that would pull an updated dependency.

dannyjdev commented:
> Please see if we can simply reference newer `Test.Utility` that would pull an updated dependency.

Updated only the test.utility version, and changed tests since error logs changed on the following PR https://github.com/NuGet/NuGet.Client/pull/3992/files.
